src/SUFST/src/can_unpack.c

```diff
@@ -5,6 +5,7 @@
 #include "usart.h"
 #include "rtcan.h"
 #include "can.h"
+#include "config.h"
 
```

Comment:
Try to disable rtcan functions in `unpack_init()` like `rtcan_init` , `rtcan_subscribe` and `rtcan_start`.
Also we won't use `rtcan_msg_consumed()` in `queue_receive_thread_entry()`